analytics/terraform/datahub-on-eks/variables.tf

```diff
@@ -23,11 +23,29 @@ variable "tags" {
 }
 
 variable "vpc_cidr" {
-  description = "VPC CIDR"
+  description = "VPC CIDR - for existing or new VPC"
   default     = "10.1.0.0/16"
   type        = string
 }
 
+variable "create_vpc" {
+  description = "Create VPC"
+  default     = true
+  type        = bool
+}
+
+variable "vpc_id" {
+  description = "VPC Id - with create_vpc set to false"
+  default     = "vpc-0bcc567a949321822"
+  type        = string
+}
+
+variable "private_subnet_ids" {
+  description = "Private subnet ids - needed with create_vpc set to false"
+  default     = ["subnet-066c6579a95470d55", "subnet-0b4bf52219857602f"]
+  type        = list(string)
+}
+
 variable "enable_vpc_endpoints" {
```

Comment:
You can remove VPC endpoints from the VPC modules if it's not used. I am thinking of removing from other blueprints as well.

Reply:
I had fixed the vpc_endpoint code so it will work if customer like to set up vpc endpoints.